ui/src/app/views/account-creation-modals/component.tsx

```diff
@@ -0,0 +1,231 @@
+import {Component, DoCheck, Input, OnInit} from '@angular/core';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {fullUrl, handleErrors} from 'app/utils/fetch';
+
+import {
+  FetchArgs,
+  ProfileApiFetchParamCreator,
+  ResendWelcomeEmailRequest,
+  UpdateContactEmailRequest
+} from 'generated/fetch/api';
+
+import {
+  Error,
+  FieldInput,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalTitle,
+  styles,
+} from 'app/react-components/modals';
+
+function isBlank(s: string) {
+  return (!s || /^\s*$/.test(s));
+}
+
+class AccountCreationResendModalReact extends React.Component<any, any> {
+
+  state: {};
+  props: {
```

Comment:
Nit: Per discussion yesterday, this should be defined above and typed in the React.Component typings.